scripts/index.py

```diff
@@ -2,23 +2,23 @@
 """Read services from the API endpoint and write to search-api for indexing.
 
 Usage:
-    index-services.py <stage> --index=<index> --frameworks=<frameworks> [options]
+    index.py <doc-type> <stage> [--create] --index=<index> --frameworks=<frameworks> --mapping=<mapping> [options]
```

Comment:
`index-to-search-api.py`? Index can mean many things across multiple services.

Reply:
Dunno, right now there is only one way to index these things, and so it's speculation as to whether _if_ our services had multiple places to be indexed, doing that would be a _separate_ script?

Reply:
Well, I mean `index` doesn't just mean search api. Also used to mean postgres indexes. Or index as in "generate an index page". Or index as in `list.index()`. Just a very generic term. Not quite as bad as `type` but getting there.